src/powershell/en-US/FinOpsToolkit.strings.psd1

```diff
@@ -10,7 +10,10 @@ ConvertFrom-StringData -StringData @'
     NewDirectory = Creating directory '{0}'.
     TemplateNotFound = Could not find template 'main.bicep' at path '{0}'.
     ContextNotFound = Could not retrieve Az context. Run Az-Login.
+    RegisterProvider =  Registering resource provider {0}.
+    ResourceProviderExists =  Resource provider {0} is already registered. 
```

Comment:
nit: `Registered` would probably be more accurate. (Don't forget to change the reference in the script.)
```suggestion
    ResourceProviderRegistered =  Resource provider {0} is already registered. 
```

Reply:
That's done, here and in script as well.